course_content/notebooks/matplotlib_intro.ipynb

```diff
@@ -281,7 +267,65 @@
       "\n",
       "## Multiple axes on the same figure (aka subplot)\n",
       "\n",
-      "Matplotlib makes it relatively easy to add more than one Axes artist to a figure. The ``add_subplot`` method on a Figure artist, which is wrapped by the ``subplot`` function in ``pyplot``, adds an Axes artist in the grid position specified. To compute the position, we must tell matplotlib the number of rows and columns to separate the figure into, and which number the axes to be created is (1 based). For example, to create axes at the top right and bottom left of a $3 x 2$ notional grid of Axes artists the grid specifications would be ``2, 3, 3`` and ``2, 3, 4`` respectively:"
+      "Matplotlib makes it relatively easy to add more than one Axes object to a figure. The ``add_subplot`` method on a Figure object, which is wrapped by the ``subplot`` function in ``pyplot``, adds an Axes object in the grid position specified. To compute the position, we must tell matplotlib the number of rows and columns (respectively) to separate the figure into, followed by the index of the axes to be created (1 based). \n",
+      "\n",
+      "For example, to create an axes grid with one row and two columns, with a plot on the left-hand side, the grid specification would be ``(1, 2, 1)``."
```

Comment:
I dislike this simplification. The original clearly demonstrated what the outcome of subplots was - that is, where the subplot axes artists would end up given some arbitrary input numbers. A lot of that is lost by placing the figure at the top left, as it is much harder to see how the input numbers relate to the output location.